backend/models/orgUser.js

```diff
@@ -11,7 +11,6 @@ const OrgUserSchema = new mongoose.Schema({
   email: {
     type: String,
     required: [true, 'required'],
-    // unique: [true, 'This email is already in use'],
     validate: [validateEmail, "please use a valid email address"],
   },
   charityNumber: String,
```

Comment:
UK Charity number - are all numbers so instead of String, we should have `Number` as schema type